src/openfermion/hamiltonians/_molecular_data.py

```diff
@@ -359,6 +376,18 @@ def canonical_orbitals(self, value):
         self._canonical_orbitals = value
 
     @property
+    def overlap_integrals(self):
```

Comment:
I know what you are doing here but because decorators are an "advanced" aspect of the language it is a good practice to add a one-liner saying what you are doing since this will confuse more novice developers.

Reply:
I'm not sure we want to add one for all 10 getters and setters, but I'll put a block comment at the top of the section.